test_data/thipe-ayeye/templates/thipe-ayeye-subobject-template.html

```diff
@@ -3,17 +3,22 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>{{ title }}</title>
+
     {# Facebook sharing image (Open Graph) #}
-    <meta property="og:url" content="{{ url }}" />
     <meta property="og:type" content="website" />
-        <meta property="og:title" content="{{ data.ids[data.entryPoint].props['http://schema.org/name'].fwd[0].value }}" />
-        <meta property="og:description" content="{{ data.ids[data.entryPoint].props.sentence.fwd[0].value }}" />
-        <meta name="description" content="{{ data.ids[data.entryPoint].props.sentence.fwd[0].value }}" />
-    {% if data.ids[data.entryPoint].props["http://schema.org/hasPart"] %}
+    <meta property="og:title" content="{{ data.ids[data.entryPoint].props['http://schema.org/name'].fwd[0].value }}" />
+    <meta property="og:description" content="{{ data.ids[data.entryPoint].props.sentence.fwd[0].value }}" />
```

Comment:
This didn't work for me. Try replacing
`data.ids[data.entryPoint].props.sentence.fwd[0].value` with 
`data.ids[data.entryPoint].props['arcp://name,custom/terms#sentence'].fwd[0].value`

similar for ln 11 below